src/fiddle.ts

```diff
@@ -12,12 +12,16 @@ function hashString(str: string): string {
   return md5sum.digest('hex');
 }
 
+/** This class represents a fiddle */
 export class Fiddle {
   constructor(
+    /** It serves as the entry point or the primary script file for the fiddle */
     public readonly mainPath: string, // /path/to/main.js
+    /** It represents the code */
```

Comment:
`source` is the source of the fiddle (URL, file path, etc)

Reply:
Done 👍